environments/ithc.yml

```diff
@@ -138,9 +138,15 @@ cname:
   - name: "afdverify.paybubble"
     ttl: 3600
     record: "afdverify.hmcts-ithc.azurefd.net"
+  - name: "fact"
```

Comment:
why do you need this? can't you have it in private dns?

Reply:
penetration-tester laptop is not permitted to access the HMCTS VPN, so the domain needs to be public